_posts/2020-03-18-celebrating-tf-dev-summit.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Celebrating #TFDevSummit 2020"
+date:   2020-03-18 13:00:00
+categories: Commentary and announcements
+highlight: true
+author: Sayak Paul
+image: https://github.com/tfugkol/tfugkol.github.io/blob/devsummit-blog/assets/img/post_images/media_devsummit_blog/image6.png
```

Comment:
Please use this form of addressing: `/assets/img/post_images/image_name.ext`

Reply:
Why?

Reply:
in case we ever move the domain in future, this will help us migrate seamlessly

Also, will help in case Github decides to change the way they serve static files

Reply:
See if it works now,